### PR TITLE
Update for bevy 0.13

### PR DESCRIPTION
**Pull request by stargazing-dino**
Mostly simple changes. The odd stuff was the lighting. Bevy got [new exposure settings](https://bevyengine.org/news/bevy-0-13/#camera-exposure) so the scene became suuuper dark. Had to massively ramp up those values. Also used the new macro from #88 in the farm sim. 

* related #92

### CODE REVIEW

File: examples/farming_sim.rs [hunk 1]
```diff
@@ -336,22 +336,20 @@ pub fn sell_need_scorer_system(
...
 // moving to a location. It's not an AI component, but a standard Bevy component
-#[derive(Clone, Component, Debug)]
+#[derive(Debug, Clone, Component, ActionBuilder)]
+#[action_label = "MyGenericLabel"]
 pub struct MoveToNearest<T: Component + std::fmt::Debug + Clone> {
-    _marker: std::marker::PhantomData<T>,
-    speed: f32,
+    // We use a PhantomData to store the type of the component we're moving to.
+    pub _marker: std::marker::PhantomData<T>,
+    pub speed: f32,
```
Comment (zkat) [inline examples/farming_sim.rs:344]: these don't need to be public and, in fact, you usually don't want `PhantomData` to be `pub`.

File: examples/farming_sim.rs [hunk 2]
```diff
@@ -483,15 +481,15 @@ fn init_entities(
 
     commands.insert_resource(AmbientLight {
         color: Color::WHITE,
-        brightness: 1.0,
+        brightness: 700.0,
```
Comment (zkat) [inline examples/farming_sim.rs:484]: I'm a little put off by this massive value, and same with `intensity`. You said that Bevy got some new lighting settings: is there some other setting on the camera or something that we need to adjust in order for `AmbientLight` and `Spotlight` to have reasonable-looking values? This just feels like it's papering over a different issue.

Comment (stargazing-dino) [reply examples/farming_sim.rs:484]: Definitely agree with the weirdness. However I remember seeing some conversation about this in rendering-dev and consensus that the "realistic" values should be much higher

Examples in the bevy repo also seem to have similarly large values:

[animated_fox](https://github.com/bevyengine/bevy/blob/a7aee1d03ac2070ae21b5f26253c0d8dc9d9af67/examples/animation/animated_fox.rs#L10) - ambient light intensity 2000.0

[alien cake addict](https://github.com/bevyengine/bevy/blob/a7aee1d03ac2070ae21b5f26253c0d8dc9d9af67/examples/games/alien_cake_addict.rs#L118) - point light intensity 2,000,000

[3d_shapes](https://github.com/bevyengine/bevy/blob/a7aee1d03ac2070ae21b5f26253c0d8dc9d9af67/examples/3d/3d_shapes.rs#L70) - point light intensity 10,000,000

File: examples/farming_sim.rs (hunk 1 above)
Comment (stargazing-dino) [reply examples/farming_sim.rs]: done

File: examples/farming_sim.rs (hunk 2 above)
Comment (zkat) [reply examples/farming_sim.rs:484]: that's so weird.

Ok, I guess.

File: examples/farming_sim.rs [hunk 3]
```diff
@@ -336,22 +336,20 @@ pub fn sell_need_scorer_system(
...
 
-// We cannot use the derive macro here because we need to be generic over 'T'.
-// Instead we manually implement the ActionBuilder trait.
-impl<T> ActionBuilder for MoveToNearest<T>
-where
-    T: Component + std::fmt::Debug + Clone,
-{
-    // An action builder needs to implement this method. It's used to attach
-    // the Action component to the actor entity.
-    fn build(&self, cmd: &mut Commands, action: Entity, _actor: Entity) {
```
Comment (ekalosak) [inline examples/farming_sim.rs:353]: Removing this I think broke the ActionBuilder derive.

**Pull request merged**